src/renderer/components/ft-profile-edit/ft-profile-edit.css

```diff
@@ -43,3 +45,43 @@
     width: 90%;
   }
 }
+
+.dropbox {
+  outline: 2px dashed grey; /* the dash box */
+  outline-offset: -10px;
+  background: lightcyan;
+  color: dimgray;
+  padding: 10px 10px;
+  min-height: 200px; /* minimum height */
+  position: relative;
+  cursor: pointer;
+}
+
+.input-file {
+  opacity: 0; /* invisible but it's there! */
+  width: 100%;
+  height: 200px;
+  position: absolute;
+  cursor: pointer;
+}
+
+.dropbox:hover {
+  background: lightblue; /* when mouse over to the drop zone, change color */
+}
+
+.dropbox p {
+  font-size: 1.2em;
+  text-align: center;
+  padding: 50px 0;
+}
+
+.default-upload {
+  position: absolute;
+  z-index: 1000;
+  opacity: 0;
+  cursor: pointer;
+  right: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+}
```

Comment:
There should be a final new line as defined in [`.editorconfig`](https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig#L9)

See https://editorconfig.org/#download for how to support `.editorconfig`